public/getting-started/index.html

```diff
@@ -0,0 +1,260 @@
+          </li>
+          <li>Click the <code>Fork</code> button.
+             <div><a href="fork.png" target="_blank"><img src="fork.png" width="800px"></a></div>
+             This puts a copy of the treebank-template repository into your own
+             GitHub account.
+          </li>
+        </ol>
+      </li>
+      <li><b>Setup GitHub Actions</b></li>
+        <ol>
+          <li>Immediately after forking the repository, you should be prompted
+            by GitHub to enable actions on your repository.
+            Click <code>Set Up Actions</code>.
+             <div><a href="forked.png" target="_blank"><img src="forked.png" width="800px"></a></div>
+          </li>
+          <li>(If you aren't prompted, click on the <code>Actions</code> tab
+            yourself anyway.)</li>
+          <li>Click <code>I understand my workflows, go ahead and enable them</code>.
+             <div><a href="actions-enable.png" target="_blank"><img src="actions-enable.png" width="800px"></a></div>
+             This step is necessary in order for GitHub to update your GitHub Pages
+             site automatically whenever you update or add files to your repository.
+             For more information on GitHub actions see <a href="https://help.github.com/en/actions"
+             target="_blank">GitHub Actions Help</a>.
+          </li>
+        </ol>
+      </li>
+      <li><b>Update the home page link for your site</b>
+        <ol>
+          <li>Click the <code>package.json</code> file in the root directory of
+            the repository.
+             <div><a href="package.png" target="_blank"><img src="package.png" width="800px"></a></div>
+          </li>
+          <li>Click the <code>edit</code> icon.
+             <div><a href="edit.png" target="_blank"><img src="edit.png" width="800px"></a></div>
+          </li>
+          <li>Replace <code>perseids-publications.github.io</code> with, e.g.,
+            <code>yourgithubaccount.github.io</code>.
+            That is, if your GitHub user name is "janedoe" you would change
+            <code>https://perseids-publications.github.io/treebank-template/</code> to
+            <code>https://janedoe.github.io/treebank-template/</code>
+             <div><a href="homepage.png" target="_blank"><img src="homepage.png" width="800px"></a></div>
+          </li>
+          <li>Scroll to the bottom of the page to <code>Commit Changes</code>
+            and enter a message to describe your
+            change to this file. (e.g. something like "Updated homepage link")
+          </li>
+          <li>Make sure the option <code>Commit directly to the master branch</code>
+            is selected
+             <div><a href="commit.png" target="_blank"><img src="commit.png" width="800px"></a></div>
+          </li>
+          <li>Click <code>Commit Changes</code></li>
+        </ol>
+      </li>
+      <li><b>Setup GitHub Pages</b>
```

Comment:
Here and in "Setup GitHub Actions" above: this is something I get confused about and could be wrong about, but I think "setup" (one word) is used as a noun and the verb is usually written "set up" (two words).

Reply:
Yes, you're right. I'll fix that.